WORKING_GROUPS.md

```diff
@@ -12,61 +12,19 @@ Groups are not formed to *begin* a set of tasks but instead are formed
 once that work is already underway and the contributors
 think it would benefit from being done as an autonomous project.
 
-If the work defined in a Working Group charter is completed, the Working Group
-should be dissolved and the responsibility for governance absorbed back into
-the TSC. A Working Group can be dissolved either through consensus of the
-Working Group membership or normal TSC motion and vote.
+If the work defined in a Working Group's charter is complete, the charter
+should be revoked.
 
-## Process:
+A Working Group's charter can be revoked either by consensus of the Working
+Group's members or by a TSC vote. Once revoked, any future work that arises
+becomes the responsibility of the TSC.
 
-* [Starting a Working Group](#starting-a-wg)
-* [Bootstrap Governance](#bootstrap-governance)
+## Joining a WG
 
-## Current Working Groups
-
-NOTE: Technical Working Groups exist in the [nodejs/ctc repository](https://github.com/nodejs/ctc/blob/master/WORKING_GROUPS.md).
-
-## De-chartered Working Groups
-
-* [Inclusivity](#inclusivity)
-
-### [Inclusivity](https://github.com/nodejs/inclusivity)
-
-*Note*: The efforts and mission of the Inclusivity Working Group have been
-assumed by the Node.js Foundation's [Community Committee][].
-
-<!--
-This information should mostly mirror:
-https://github.com/nodejs/inclusivity#statement-of-purpose
--->
-
-The Inclusivity Working Group seeks to increase inclusivity and diversity
-for the Node.js project:
-  * *Increasing inclusivity* means making the Node.js project a safe and
-    friendly place for people from diverse backgrounds.
-  * *Increasing diversity* means actively onboarding people from diverse
-    backgrounds to the Node.js project and maintaining their participation.
+To find out how to join a working group, consult the GOVERNANCE.md in
+the working group's repository, or in the working group's repository.
```

Comment:
This is sentence is definitely messed up. I'm not sure what was intended. But `in the working group's respository, or in the working group's repository` has to be some kind of copy/paste error?